src/network.cpp

```diff
@@ -102,12 +106,18 @@ void setupWebServer() {
                       BATTERYVALUES_HTML_SIZE, templateProcessor);
       return;
     case HTTP_POST:
-      const auto scaleParam = request->getParam("c", true);
-      if (scaleParam == nullptr || scaleParam->value().length() < 1) {
-        request->send(200, "text/html", "Invalid scaling parameter.");
+      const auto coefficient = request->getParam("coefficient", true);
+      const auto offset = request->getParam("offset", true);
+      if (coefficient == nullptr || coefficient->value().length() < 1) {
```

Comment:
No need for length checking here, float conversion should be taking care of this.